File: 240122/학점 계산기/credit-calculator.py
n = int(input())
sum_val = 0

arr = list(map(float, input().split()))
a = len(arr)
for j in range(a):
    sum_val += arr[j]
        
    div_val = sum_val / n
print('%0.1f' %div_val)

if div_val >= 4.0:
    print('Perfect')
elif div_val >= 3.0 :
    print('Good')
else :
    print('Poor')

File: 240921/학점 계산기/credit-calculator.cpp
#include <iostream>
using namespace std;

int main() {
    // 여기에 코드를 작성해주세요.
    int n;
    cin >> n;
    double sum = 0;
    int cnt = 0;
    double arr[100];
    for (int i = 0; i < n; i ++) {
        cin >> arr[i];
        sum += arr[i];
        cnt ++;
    }
    cout<< fixed;
    cout.precision(1);
    cout << sum / cnt << endl;
    if (sum / cnt >= 4.0) {
        cout << "Perfect";
    }
    else if (sum / cnt >= 3.0) {
        cout << "Good";
    }
    else {
        cout << "Poor";
    }
    return 0;
}